### PR TITLE
🚑(frontend) preserve query string on language change

### PR DESCRIPTION
**Pull request by jbpenrath**
## Purpose

When user switches to another language we want to preserve the query string.

## Proposal

- [x] Update LanguageSelector component to append `location.search` to selectedItem url

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unrealeased]
 
+### Fixed
+
```
Comment (sampaccoud) [inline CHANGELOG.md:12]: On your commit title: s/change/switch

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
...
+- Preserve query string on language change
```
Comment (sampaccoud) [inline CHANGELOG.md:13]:
```suggestion
- Preserve query string on language switch
```

File: src/frontend/js/components/LanguageSelector/index.spec.tsx [hunk 3]
```diff
@@ -41,4 +49,35 @@ describe('<LanguageSelector />', () => {
       name: 'Switch to Français (currently selected)',
     });
   });
+
+  it('should preserve location search if there are', async () => {
```
Comment (sampaccoud) [inline src/frontend/js/components/LanguageSelector/index.spec.tsx:53]:
```suggestion
  it('should preserve location search parameters if there are any', async () => {
```

File: src/frontend/js/components/LanguageSelector/index.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import { useSelect } from 'downshift';
 import { FC } from 'react';
-import { defineMessages, useIntl, FormattedMessage } from 'react-intl';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
```
Comment (sampaccoud) [inline src/frontend/js/components/LanguageSelector/index.tsx:3]: Ah ! Linter issue this time? 😛

**Review by sampaccoud — APPROVED** [review]

File: src/frontend/js/components/LanguageSelector/index.tsx (hunk 4 above)
Comment (jbpenrath) [reply src/frontend/js/components/LanguageSelector/index.tsx:3]: IDE linter 😅

**Pull request merged**